lib/content-services/src/lib/tag/tags-creator/tags-creator.component.html

```diff
@@ -1,54 +1,40 @@
 <div class="adf-tags-creation">
+    <div *ngIf="isNameTagsVisible" class="adf-tag-name-field">
+        <mat-form-field class="adf-tag-search-field" appearance="fill">
+            <input #tagNameInput
+                    matInput 
+                    autocomplete="off"
+                    [formControl]="tagNameControl"
+                    (keyup.enter)="addTag()"
+                    adf-auto-focus
+                    placeholder="{{'TAG.TAGS_CREATOR.INPUT_PLACEHOLDER' | translate}}" 
+            />
+            <mat-error *ngIf="tagNameControl.invalid">{{ tagNameErrorMessageKey | translate }}</mat-error>
+        </mat-form-field >
+    </div>
     <p
        class="adf-no-tags-message"
-       *ngIf="!tags.length && !tagNameControlVisible">
+       *ngIf="!isTagsEmpty">
```

Comment:
incorrect check, the "empty" screen should be displayed when "isTagsEmpty" is true